### PR TITLE
Adds badges and mentions Types in the README

### PR DESCRIPTION
**Pull request by bravo-kernel**

### CODE REVIEW

**Comment by bravo-kernel** [comment]
I hope the ``Type`` descriptions make sense, let me know if they aren't

File: README.md [hunk 1]
```diff
@@ -107,3 +113,15 @@ could filter your articles using the following.
 Would filter your list of articles to any article with "cakephp" in the `title`
 or `content` field. You might choose to make a `get` form which posts the filter
 directly to the url, or create links manually.
+
+## Types
+
+The Search plugin comes with a set of predefined ``Types`` that allow you to
+easily create the search results you need. Use:
```
Comment (ADmad) [inline README.md]: "..search filters.." would be more apt imo

**Comment by bravo-kernel** [comment]
Changed to search filters

**Comment by ADmad** [comment]
@bravo-kernel Thanks. The tests could also use some love if you can spare some time, current 61% doesn't look too nice :)

**Comment by bravo-kernel** [comment]
np, I can always remove the badge :wink:

**Comment by ADmad** [comment]
Or we can show our short comings proudly :stuck_out_tongue:

**Pull request merged**